frontend/src/types/ExperimentCollection.ts

```diff
@@ -1,14 +1,10 @@
 import Experiment from "./Experiment";
 
 export default interface ExperimentCollection {
-    id: number;
     slug: string;
     name: string;
     description: string;
     dashboard: Experiment[];
```

Comment:
Looking at the code this might be a `null` value or even omitted completely from the object - which I think is a bad practice, as mentioned before. But if you're going to use it, this should probably be:

Assuming an explicit `null` value.
```suggestion
    dashboard: Experiment[] | null;
```

or, when it is omitted from the json body:
```suggestion
    dashboard?: Experiment[];
```

Reply:
As per your suggestion, the value returned if `dashboard=False` is a `[]`, not `None`, so no `null` value needed here.